src/client/pages/VergunningDetail/EigenParkeerplaats.tsx

```diff
@@ -11,70 +16,84 @@ export function EigenParkeerplaats({
 }) {
   const isVerleend = vergunning.processed && vergunning.decision === 'Verleend';
   const isAfgehandeld = vergunning.processed;
-  let location1 = null;
-  let location2 = null;
+  const locations: ReactNode[][] = [[], []];
 
   if (Array.isArray(vergunning.locations)) {
-    [location1, location2] = vergunning.locations;
+    vergunning.locations.forEach((location, i) => {
```

Comment:
Je kunt dit evt ook wat netter oplossen door:

1. Niet te loopen en het zoals het was alleen de grouping toe te voegen.
2. Er een Apart componentje van te maken (preferred)

```
function InfoDetailLocation({ adresLabel = 'Adres', location }: ParkeerPlaats & { adresLabel: string }) {
  return (
    <>
      <Location
        label={adresLabel}
        location={`${location.street} ${location.houseNumber}`}
      />
      <InfoDetail label="Soortplek" value={location.type} />
      <InfoDetail label="Parkeervak" value={location.url} />
    </>
  );
}
```
En vervolgens:
```
<InfoDetailGroup>
      {!!location1 && <div>
        <InfoDetailLocation location={location1} />
      </div>}
      {!!location2 && <div className={styles2.InfoGroupDivider}>
        <InfoDetailLocation adresLabel="Adres 2" location={location2} />
      </div>}
</InfoDetailGroup>
```

of

```
const locations = vergunning.locations((location, i) => {
   const adresLabel = `Adres ${i>0 ? i+1 : ''}`
    return <div className={i>0&&styles2.InfoGroupDivider}><InfoDetailLocation adresLabel={adresLabel} location={location} /></div>
});

<InfoDetailGroup>
  {locations}
</InfoDetailGroup>
```

of 3.
```
const locations = vergunning.locations.map((location, i) => {
    const adresLabel = `Adres ${i>0 ? i+1 : ''}` // of const adresLabel = i>0? `Adres ${i+1}` : 'Adres'; 
    return (
      <div className={i>0&&styles2.InfoGroupDivider}>
        <Location
          label={adresLabel}
          location={`${location.street} ${location.houseNumber}`}
        />
        <InfoDetail label="Soortplek" value={location.type} />
        <InfoDetail label="Parkeervak" value={location.url} />
      </div>
    );
  });
  
<InfoDetailGroup>
  {locations}
</InfoDetailGroup>
```

Reply:
(untested)